include/control_filters/low_pass_filter.hpp

```diff
@@ -0,0 +1,182 @@
+// Copyright (c) 2021, Stogl Robotics Consulting UG (haftungsbeschränkt)
```

Comment:
Let's bump the year.

```suggestion
// Copyright (c) 2023, Stogl Robotics Consulting UG (haftungsbeschränkt)
```